resources/views/elephpant/index.blade.php

```diff
@@ -12,6 +12,9 @@
 </div>
 <div class="container">
     @include('partials._search')
+    <p class="mb-4">
+        ElePHPants marked as <strong>Prototype Only</strong> are for reference and cannot be added to your herd or traded with other users as they were never mass produced.
```

Comment:
This disclaimer is less important than the search results so I think it should come after the results.  Also, there is no need to show this text when there are no results, so this could go between the `@endforeach` and `@else`